style.css

```diff
@@ -496,3 +501,605 @@ button:active {
+    width: 100%;
+    background-color: #e5e5e5;
+    padding-left: 142px;
+    padding-right: 142px;
+  }
+
+  .card {
+    width: 100%;
+    height: 496px;
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    border-radius: 16px;
+    margin-top: 44px;
+    padding: 16px;
+    margin-bottom: 24px;
+    border: 2px solid #e5e5e5;
+    background-color: #fff;
+  }
+
+  #card1,
+  #card3 {
+    display: flex;
+  }
+
+  #card2,
+  #card4 {
+    flex-direction: row-reverse;
+  }
+
+  .desktop-card-image {
+    display: flex;
+    width: 100%;
+    height: 448px;
+    margin: 23px;
+  }
+
+  .desktop-card-content {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 448px;
+    margin: 23px;
+  }
+
+  .image1 {
+    display: block;
+  }
+
+  .card-head {
+    display: flex;
+    box-sizing: border-box;
+    justify-items: flex-start;
+    height: 32px;
+    width: 297px;
+    padding-left: 0;
+    margin-top: 0;
+  }
+
+  h3 {
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 44px;
+    text-align: left;
+    color: #172b4d;
+    margin-top: 12px;
+    margin-bottom: 12px;
+  }
+
+  .tonic {
+    padding-left: 0;
+    list-style: none;
+  }
+
+  .headline1 {
+    height: 80px;
+    width: 100%;
+  }
+
+  .headline2 {
+    width: 496px;
+    height: 46px;
+    text-align: left;
+  }
+
+  .break {
+    display: none;
+  }
+
+  .canopy1 {
+    list-style: none;
+    height: 16px;
+    width: auto;
+    left: 0;
+    font-size: 13px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 16px;
+    color: #344563;
+  }
+
+  .canopy {
+    font-size: 13px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  .cardsub {
+    list-style: none;
+    display: flex;
+    padding-left: 0;
+    justify-content: space-between;
+    justify-self: left;
+    height: 24px;
+    width: 203px;
+  }
+
+  .list {
+    box-sizing: border-box;
+    background-color: #ebebff;
+    color: #6070ff;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 16px;
+    border-radius: 8px;
+    justify-content: space-between;
+  }
+
+  p {
+    height: 50px;
+    width: 490px;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #344563;
+    margin-top: 24px;
+  }
+
+  .c2 {
+    margin-top: 48px;
+  }
+
+  button {
+    height: 48px;
+    width: 124px;
+    box-sizing: border-box;
+    font-size: 17px;
+    color: #396df2;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 24px;
+    border-radius: 8px;
+    padding: 12px;
+    border: 1px solid #6070ff;
+    margin-top: 12px;
+    background-color: #fff;
+  }
+
+  button:hover {
+    color: #fff;
+    background-color: #6070ff;
+
+    /* transition-property: all;
+    transition: 0.2s ease-in-out; */
+  }
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly fix it. :writing_hand: